src/geo/transform.js

```diff
@@ -385,6 +385,47 @@ class Transform {
             this.zoom)._zoomTo(zoom);
     }
 
+    /**
+     * Get the latitude and longitude under the camera, the altitude of the camera in meters, bearing & pitch
+     * @returns {Object} containing longitude, latitude, altitude and pitch(in degrees)
+     */
+    getCameraPosition(): { lng: number, lat: number, altitude: number, pitch: number, bearing: number } {
+        const pitch = this._pitch;
+        const altitude = Math.cos(pitch) * this.cameraToCenterDistance;
+        const latOffset = Math.tan(pitch) * this.cameraToCenterDistance;
+        const latPosPointInPixels = this.centerPoint.add(new Point(0, latOffset));
+        const latLong = this.pointLocation(latPosPointInPixels);
```

Comment:
Let's call this `lngLat` to correspond to our naming conventions elsewhere.

Reply:
fixed